home/tests/test_insert_question_feature.py

```diff
@@ -110,9 +110,12 @@ def test_post_valid_question_with_client(self, client, valid_data):
                 assert Question.objects.get(title=valid_data["title"])
 
             client.login(username='Lior', password='LiorLior')
```

Comment:
you can use the fixture authenticated_user instead of preform client.login in this and in the next test function.
you dont need to import anything because its in the conftest.py file, just add a parameter to your function : 
authenticated_user and the client will be already logged in. like in Answer feature tests right [here](https://github.com/beyond-io/shlifim/pull/87/files#diff-f7c34089a18939177812719101f6af75ca1bd7ed354d3fc4bf8ea6509985bc89R17)

Reply:
Thanks for the offer. 
I can do it, but I didn't understand why should I prefer that suggestion instead of what I did?

Reply:
I agree with Ido, if you have a fixture that does the job for you why do it manually again? 
The whole point of conftest.py was to enable us to use in fixtures in different test files.

Reply:
Ok, added